### PR TITLE
Add: First program

### PR DESCRIPTION
**Pull request by Sudarsan-Pradhan**
I added a java program which will check an entered number given by the user is a palindrome number or not

### CODE REVIEW

**Review by Dev-Soumyaranjan — CHANGES_REQUESTED** [review]
This is looking ✨!! I have left a few comments that should be addressed before this gets merged🚀

File: PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+//Wrtite a java program to check whether an entered number is a palindrom or not
+
+//Importing scanner class inorder to use nextInt() method for taking input from user
+import java.util.Scanner;
+
+/*
+* We are given an integer input as the number, the objective is to check whether or not the given number is a palindrome.
+* To do so, we’ll first reverse the string input using loops and check if it matches the original number.
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java]: Change "string" to "number" at line number 8

File: PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* Output : This is not a palindrome number
+*
+* */
+public class Check_Palindrome {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter a number:");
+        int number = input.nextInt();// Taking input from the user
+        // If isPalindrome() method returns true then the number is palindrome and if it
+        // returns false then it is not
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java]: Remove the comment line 28 and merge it to the line number 27 and make it single comment line.

File: PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+         * after while loop number will be zero(because while loop will continue until
+         * and unless number is zero)
+         * But after while loop we need to check reverse number with original number
+         * That's why first we store the number in temp variable so that after
+         * completion of while loop
+         * we can compare temp with reverse number
+         */
+        int temp = number;
+        // Initializing rev variable with zero and after while loop the reverse of the
+        // number will store in rev
```
Comment (Dev-Soumyaranjan) [inline PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java]: Remove the comment line 53 and merge it to the line number 52 and make it single comment line.

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @Sudarsan-Pradhan  for adding new code

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @Sudarsan-Pradhan for code

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @Sudarsan-Pradhan for code

**Comment by Sudarsan-Pradhan** [comment]
i have changed the code accordingly. just check it out

**Review by Dev-Soumyaranjan — APPROVED** [review]
Sounds good✨Thanks for contributing. I am going to merge it

**Comment by Sudarsan-Pradhan** [comment]
Thank you very much

**Comment by Dev-Soumyaranjan** [comment]
https://github.com/all-contributors please add @Sudarsan-Pradhan for code

**Comment by Dev-Soumyaranjan** [comment]
@all-contributors please add @Sudarsan-Pradhan for code

**Pull request merged**